docs/blog/newsletter-may-2024.rst

```diff
@@ -0,0 +1,120 @@
+:og:image: https://www.writethedocs.org/_static/logo-opengraph.png
+
+.. post:: May 07, 2024
+    :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – May 2024
+#########################################
+
+Hello once again, fellow documentarians. I hope all is well with you and yours.
+
+Our online Atlantic 2024 conference is planned for September and there's still time to get in your proposal to speak. Check out the `Call for Proposals </conf/atlantic/2024/cfp/>`__ and get your ideas in by May 15. The Atlantic 2024 conference welcomes speakers from around the world and you don't even have to travel to give the talk. So if you've ever considered giving a talk before, send in your proposal and give it a try.
+
+If you'd like some inspiration from past speakers, the Portland 2024 conference wrapped up since our last newsletter. See the `conference recap </conf/portland/2024/news/thanks-recap/>`__ and see `videos of all the talks <https://www.youtube.com/playlist?list=PLZAeFn6dfHpm4FboYSTD1Bs8Yp8k_JvAL>`__ to get a taste of what the conference was like. And then get inspired and `submit a talk for Atlantic 2024 </conf/atlantic/2024/cfp/>`__!
+
+We'd also like to announce the Australia 2024 conference dates. This year, the conference will take place in Melbourne on 28 and 29 November. We'll share more details around venue, tickets, and the CFP process in a couple of weeks.
```

Comment:
@swapnilogale @CollierCZ are we nixing this entirely? Or maybe rewording to a more vague "finishing touches etc"

Reply:
Happy to replace 28 and 29 with "late November". Rest of the text sounds fine.

Reply:
```suggestion
We'd also like to announce the Australia 2024 conference dates. This year, the conference will take place in Melbourne in late November. We'll share more details around venue, tickets, and the CFP process in a couple of weeks.
```